.github/workflows/format.yml

```diff
@@ -4,20 +4,20 @@ on:
     branches: [main]
 jobs:
   format:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Any reason for the fixed version other than just using latest?

Reply:
While running the Github Action I saw some errors. But now I see that these are just warnings, indicating that the `ubuntu-latest` tag will use `ubuntu-22.04` soon. So I believe it's possible to keep it as it was before. I will rever it in a new commit.

https://github.com/rentalhost/vscode-npm-outdated/actions/runs/3670179512

> **ubuntu-latest** workflows will use **ubuntu-22.04** soon. For more details, see https://github.com/actions/runner-images/issues/6399